common/local_utils.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper functions for using the local_utils tool."""
+
+import os
+
+from common import logs
+from common import new_process
+
+logger = logs.Logger('local_utils')
+
+
+def local_utils_command(arguments, *args, **kwargs):
+    """Executes a local_utils command with |arguments| and returns the result.
+    """
+    return new_process.execute(arguments, *args, **kwargs)
+
+
+def cp(*cp_arguments, **kwargs):  # pylint: disable=invalid-name
```

Comment:
I think at this point we should definitely change the API of `cp` to take `src` and `dst` rather than `cp_arguments`  if someone passed `-r` to `cp()` this breaks.